make.js

```diff
@@ -367,8 +367,8 @@ target.package = function() {
     // nuspec
     var version = options.version;
     if (!version) {
-        console.log('ERROR: supply version with --version');
-        process.exit(0);
+        console.warn('Skipping nupkg creation. Supply version with --version.');
+        return;
```

Comment:
Thanks eric, I am going ahead and merging the PR. Please let me know your thoughts, if any.